src/components/sidebar/sidebar.component.tsx

```diff
@@ -18,6 +18,7 @@ function Sidebar({
   setValues,
   allowUnspecifiedAll,
   defaultPage,
+  hideFormCollapseToggle,
```

Comment:
Is there a particular reason u added this to the sidebar? there no sections in the sidebar (side navigation)

Reply:
I added this to be called when the the form is cancelled, so as to hide the toggle from workspace header.